src/blockly/generators/propc.js

```diff
@@ -414,7 +414,7 @@ Blockly.propc.finish = function (code) {
         for (var idk in function_vars) {
             if (definitions[idx] === function_vars[idk] && definitions[idx].indexOf('volatile') === -1) {
                 //TODO: uncomment this when optimization is utilized!
-                if(inDemo) {
+                if(isExperimental.indexOf('volatile' > -1)) {
```

Comment:
Where are these experimental flags documented?

Reply:
Nowhere yet - what do you think would be the best place?  There are 3 - 'blocks', 'volatile', and 'saveas'.

Reply:
This might be a good time to look at populating the project wiki with some of these details. I'll evaluate that this next week.